roles/install_cluster/tasks/main.yml

```diff
@@ -210,4 +210,4 @@
   until: cluster.json.status in cluster_installation_states
   retries: 5
   delay: 60
-  
+...
```

Comment:
Note sure if there should be triple dots here. Is this a yaml thing?

Reply:
yeah this is just yaml syntax for "end of document"